src/Util/ClassSourceManipulator.php

```diff
@@ -197,8 +233,36 @@ public function addSetter(string $propertyName, $type, bool $isNullable, array $
         $this->addMethod($builder->getNode());
     }
 
-    public function addMethodBuilder(Builder\Method $methodBuilder)
+    public function addConstructor(array $params = [], string $methodBody = null)
```

Comment:
I think we could make both arguments required - I don't see a big use-case for an empty constructor body.

Reply:
You right, done. 
At the beginning it was optional because i worked on a another strategy.